docs/developers/relayed-transactions.md

```diff
@@ -253,136 +253,65 @@ Decoding the arguments ([useful resources here](/developers/sc-calls-format/)) w
 This feature is not yet available on **Mainnet**. See [Spica Protocol Upgrade](https://governance.multiversx.com/proposal/erd1qqqqqqqqqqqqqpgq4qvrwlr2e6ld50f3qfc94am38p8298kthg4s3f0vfn/1).
 :::
 
-Relayed transactions v3 feature comes with a change on the entire transaction structure, adding a new optional field `InnerTransactions`, which is a collection of inner transactions. That being said, relayed transactions v3 allow the user to send multiple inner transactions on the same relayed transaction which will be executed as normal transactions, without the gas consuming data field of the old relayed transactions versions.
+Relayed transactions v3 feature comes with a change on the entire transaction structure, adding two new optional fields: 
+- `relayer`, which is the relayer address that will pay the fees.
+- `relayerSignature`, the signature of the relayer that proves the agreement of the relayer.
 
-In terms of gas limit computation, let's consider the following example: relayed transaction with one inner transaction of type move balance, that also has a data field `test` of length 4.
+That being said, relayed transactions v3 will look and behave very similar to a regular transaction, the only difference being the gas consumption from the relayer. So the consuming data field of the old relayed transactions versions was removed.
```

Comment:
I think the phrase "So the consuming data field of the old relayed transactions versions was removed." is not very clear.

maybe rephrase to: It is no longer needed to specify the user transaction in the data field.